packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -616,6 +643,19 @@ export class TablePageObject<T extends TableRecord> {
         return cells.item(columnIndex);
     }
 
+    private getCellById(recordId: string, columnId: string): TableCell {
+        const row = this.getRowById(recordId);
+        const cell: TableCell | null = row.shadowRoot!.querySelector(
+            `nimble-table-cell[column-id="${CSS.escape(columnId)}"]`
```

Comment:
ah, excellent catch, yea the ids can be arbitrary strings. And using CSS.escape is the right escape for this case since it's going in a CSS selector. 👍

Reply:
Credit goes to Malcolm for that one!